app/controllers/api/clumps_controller.rb

```diff
@@ -0,0 +1,33 @@
+module API
+  class InvalidLocale < Exception;  end
+
+  class ClumpsController < ApplicationController
```

Comment:
Ok :+1: I am just asking because the admin is using ApplicationController. Any change on app controller will affect this api clumps controller.